src/Accounts/Authentication/Utilities/AzConfigReader.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Microsoft.Azure.Commands.Common.Authentication.Abstractions;
+using Microsoft.Azure.Commands.Shared.Config;
+using Microsoft.Azure.PowerShell.Common.Config;
+
+namespace Microsoft.Azure.Commands.Common.Authentication.Utilities
+{
+    static public class AzConfigReader
```

Comment:
make it internal & add licence header

Reply:
Done

Reply:
I still have concerns about the design of AzConfigReader. For example, under what circumstances should you add new method to this type? Whenever a new config is introduced? But we can discuss it later since changing its design won't be breaking change